index.html

```diff
@@ -138,6 +138,15 @@ <h2>Introduction</h2>
     &lt;body&gt;
     &lt;img id="image0" src="https://www.w3.org/Icons/w3c_main.png" /&gt;
     &lt;script&gt;
+    // Know when the entry types we would like to use are not supported.
+    function detectSupport(entryTypes) {
+      for (const entryType of entryTypes) {
+        if (!PerformanceObserver.supportedEntryTypes.includes(entryType)) {
+          console.log(entryType + " is not supported by PerformanceObserver!");
+        }
+      }
+    }
+    detectSupport(["resource", "mark", "measure"]);
```

Comment:
This example is correct but not necessarily very useful. Maybe show an example of "if supported, register a PO, otherwise do something else (e.g. poll the timeline)"

Reply:
Why would we poll the timeline when unsupported? If an entry type is unsupported in PerformanceObserver then it won't be buffered.

Reply:
On second thought, this is not supposed to resolve feature detection necessarily, so my example doesn't necessarily make sense, at least not in practice. Maybe instead of (or alongside) a console log, you can also turn on a flag that indicates the client-side analytics code that there's no need to way for unsupported PO callbacks.

Reply:
Ok, I've replaced the console.log with a comment.